bin/create-io-images

```diff
@@ -82,15 +82,15 @@ def create_image(config_filename):
     else:
         image_directory = tempfile.mkdtemp()
 
-        shutil.copy(os.path.join(svn_dir, 'run-job'), image_directory)
+        shutil.copy(os.path.join(vmu.RUN_DIR, 'bin', 'run-job'), image_directory)
```

Comment:
```suggestion
        shutil.copy(os.path.join(vmu.RUN_DIR, 'bin/run-job'), image_directory)
```
Don't concatenate literals with os.path.join - it makes them harder to grep for.